src/one.js

```diff
@@ -0,0 +1,6 @@
+export default (selection, name, className) =>
+  selection
+    .selectAll(`${name}.${className}`)
```

Comment:
Looking at this again, `className` should probably be made optional. It's nice sometimes to omit a class name if it's not necessary.